packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFDataConverters.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import "FWFGeneratedWebKitApis.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Converts an FWFNSUrlRequestData to an NSURLRequest.
+ *
+ * @param data The data object containing information to create an NSURLRequest.
+ *
+ * @return An NSURLRequest or nil if data could not be converted.
+ */
+id FWFConvertURLRequestData(FWFNSUrlRequestData* data);
```

Comment:
I generally suggest following Apple's Obj-C-ish C function naming convention of naming the returned type for things like this (e.g., `NSRectFromCGRect` rather than `ConvertCGRect`), so `FWFNSURLRequestFromRequestData`.